testing-common/src/main/groovy/io/opentelemetry/instrumentation/test/base/HttpServerTest.groovy

```diff
@@ -99,6 +105,10 @@ abstract class HttpServerTest<SERVER> extends InstrumentationSpecification imple
     true
   }
 
+  boolean testConcurrency() {
+    false
```

Comment:
Should it be true by default?

Reply:
This requires implementing support for the new endpoint, `INDEXED_CHILD`. Thus it cannot be default.